pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/StudyLanguageSql.java

```diff
@@ -41,4 +46,17 @@ int updateDefaultLanguage(@Bind("studyId") long studyId,
             + " where umbrella_study_id = :studyId and is_default = true")
     List<Long> selectDefaultLanguageCodeId(long studyId);
 
+    @SqlQuery("select language_code.iso_language_code as 'languageCode', "
```

Comment:
I don't think we need the single quotes. Typically, I prefer to leave the column names in snake-case and do the mapping in the Java class constructor itself using annotations.